presto-hive/src/main/java/com/facebook/presto/hive/rule/HiveFilterPushdown.java

```diff
@@ -153,6 +151,18 @@ public PlanNode optimize(PlanNode maxSubplan, ConnectorSession session, Variable
         return rewriteWith(new Rewriter(session, idAllocator), maxSubplan);
     }
 
+    // If this method returns true, the filter will not be pushed inside the TableScan node.
+    // This is exposed for dynamic or computed column functionality support.
+    // Consider the following case:
+    // Read a base page from the file. Modify the page to add a new column or update an existing column
```

Comment:
Can we avoid mentioning "page" here, since its a detail that shouldn't come up during planning. You can just mention row